content/post/AG2020.en.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "2020 General Assembly Reviews"
+date: 2020-09-06
+draft: false
+tags: [general-assembly]
+---
+
+As every year, we organize our General Assembly. On September 5th, we met in the region of Nantes, with a few remote members at [BigBlueButton](https://bigbluebutton.org/) exhibition to review the past year and decide on the orientations of the association.
```

Comment:
I'd rather use «using [BigBlueButton](https://bigbluebutton.org/) virtual meeting software» and «organisation» instead of «association»

Reply:
Something like this?
```suggestion
As every year, we organize our General Assembly. On September 5th, we met in the region of Nantes, with a few remote members using [BigBlueButton](https://bigbluebutton.org/) virtual meeting software to review the past year and decide on the orientations of the organisation.
```